docs/changelog.md

```diff
@@ -11,3 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Installation and run scripts
 - Changelog
+- Docker support
+
+### Fixed
+
+- HTML tags in sc-links
+- Production build
+- No refresh to enable SCg
```

Comment:
Is doesn't work

Reply:
What feature are you talking about now? Tested once again and 'no refresh to enable SCg' works as I expected. Give me some more context, please

Reply:
Oh, looks like here is something strange happened. This feature works in production, but not in dev environment. I'll remove this from changelog.